js/modules/k6/experimental/csv/module.go

```diff
@@ -0,0 +1,238 @@
+// Package csv provides a CSV parser for k6.
+package csv
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"io"
+	"sync/atomic"
+
+	"gopkg.in/guregu/null.v3"
+
+	"go.k6.io/k6/js/promises"
+
+	"go.k6.io/k6/js/modules/k6/experimental/fs"
+
+	"github.com/dop251/goja"
+	"go.k6.io/k6/js/common"
+	"go.k6.io/k6/js/modules"
+)
+
+// TODO: Should we have an option to cycle through the file (restart from start when EOF is reached, or done is true)?
```

Comment:
My personal opinion; if it doesn't pollute a lot the code/API, then I'd go for it, as in theory it doesn't sound like something very complex while it might be very handy for certain use cases.